### ISSUE TITLE
c++版本部署编译完成之后预测问题

### ISSUE THREAD

**Issue by wumuyu9** [PaddlePaddle/PaddleOCR#5183]
环境：虚拟机centos 7.4，（使用docker与否都会出现这种情况）cmake版本：3.16.0，opencv3.4.12，paddleocr release 2.4版本，gcc版本8.2.0，paddle_inference 是官网上的manylinux_cpu_avx_mkl_gcc8.2
使用测试的几张图片都是这样，c++编译成功时输出如下这种，而且预测的文字框会是整个图片。

mode: system
total images num: 1
--- Running analysis [ir_graph_build_pass]
--- Running analysis [ir_graph_clean_pass]
--- Running analysis [ir_analysis_pass]
--- Running IR pass [simplify_with_basic_ops_pass]
--- Running IR pass [layer_norm_fuse_pass]
---    Fused 0 subgraphs into layer_norm op.
--- Running IR pass [attention_lstm_fuse_pass]
--- Running IR pass [seqconv_eltadd_relu_fuse_pass]
--- Running IR pass [seqpool_cvm_concat_fuse_pass]
--- Running IR pass [mul_lstm_fuse_pass]
--- Running IR pass [fc_gru_fuse_pass]
---    fused 0 pairs of fc gru patterns
--- Running IR pass [mul_gru_fuse_pass]
--- Running IR pass [seq_concat_fc_fuse_pass]
--- Running IR pass [squeeze2_matmul_fuse_pass]
--- Running IR pass [reshape2_matmul_fuse_pass]
--- Running IR pass [flatten2_matmul_fuse_pass]
--- Running IR pass [map_matmul_v2_to_mul_pass]
--- Running IR pass [map_matmul_v2_to_matmul_pass]
--- Running IR pass [map_matmul_to_mul_pass]
--- Running IR pass [fc_fuse_pass]
--- Running IR pass [repeated_fc_relu_fuse_pass]
--- Running IR pass [squared_mat_sub_fuse_pass]
--- Running IR pass [conv_bn_fuse_pass]
WARNING: Logging before InitGoogleLogging() is written to STDERR
I0106 08:35:17.984140 23074 fuse_pass_base.cc:57] ---  detected 33 subgraphs
--- Running IR pass [conv_eltwiseadd_bn_fuse_pass]
--- Running IR pass [conv_transpose_bn_fuse_pass]
--- Running IR pass [conv_transpose_eltwiseadd_bn_fuse_pass]
I0106 08:35:17.996150 23074 fuse_pass_base.cc:57] ---  detected 1 subgraphs
--- Running IR pass [is_test_pass]
--- Running IR pass [runtime_context_cache_pass]
--- Running analysis [ir_params_sync_among_devices_pass]
--- Running analysis [adjust_cudnn_workspace_size_pass]
--- Running analysis [inference_op_replace_pass]
--- Running analysis [memory_optimize_pass]
I0106 08:35:18.014470 23074 memory_optimize_pass.cc:216] Cluster name : x  size: 12
I0106 08:35:18.014521 23074 memory_optimize_pass.cc:216] Cluster name : elementwise_add_7  size: 224
I0106 08:35:18.014529 23074 memory_optimize_pass.cc:216] Cluster name : elementwise_add_1  size: 64
I0106 08:35:18.014536 23074 memory_optimize_pass.cc:216] Cluster name : batch_norm_44.tmp_0  size: 1920
I0106 08:35:18.014544 23074 memory_optimize_pass.cc:216] Cluster name : conv2d_88.tmp_0  size: 1920
I0106 08:35:18.014550 23074 memory_optimize_pass.cc:216] Cluster name : batch_norm_46.tmp_3  size: 1920
I0106 08:35:18.014556 23074 memory_optimize_pass.cc:216] Cluster name : elementwise_add_3  size: 96
I0106 08:35:18.014564 23074 memory_optimize_pass.cc:216] Cluster name : batch_norm_44.tmp_3  size: 1920
I0106 08:35:18.014569 23074 memory_optimize_pass.cc:216] Cluster name : conv2d_90.tmp_0  size: 384
I0106 08:35:18.014576 23074 memory_optimize_pass.cc:216] Cluster name : conv2d_89.tmp_0  size: 384
--- Running analysis [ir_graph_to_program_pass]
I0106 08:35:18.098266 23074 analysis_predictor.cc:717] ======= optimize end =======
I0106 08:35:18.102048 23074 naive_executor.cc:98] ---  skip [feed], feed -> x
I0106 08:35:18.104975 23074 naive_executor.cc:98] ---  skip [sigmoid_0.tmp_0], fetch -> fetch
label file: ../../ppocr/utils/ppocr_keys_v1.txt
--- Running analysis [ir_graph_build_pass]
--- Running analysis [ir_graph_clean_pass]
--- Running analysis [ir_analysis_pass]
--- Running IR pass [simplify_with_basic_ops_pass]
--- Running IR pass [layer_norm_fuse_pass]
---    Fused 0 subgraphs into layer_norm op.
--- Running IR pass [attention_lstm_fuse_pass]
--- Running IR pass [seqconv_eltadd_relu_fuse_pass]
--- Running IR pass [seqpool_cvm_concat_fuse_pass]
--- Running IR pass [mul_lstm_fuse_pass]
--- Running IR pass [fc_gru_fuse_pass]
---    fused 0 pairs of fc gru patterns
--- Running IR pass [mul_gru_fuse_pass]
--- Running IR pass [seq_concat_fc_fuse_pass]
--- Running IR pass [squeeze2_matmul_fuse_pass]
--- Running IR pass [reshape2_matmul_fuse_pass]
--- Running IR pass [flatten2_matmul_fuse_pass]
--- Running IR pass [map_matmul_v2_to_mul_pass]
I0106 08:35:18.231897 23074 fuse_pass_base.cc:57] ---  detected 1 subgraphs
--- Running IR pass [map_matmul_v2_to_matmul_pass]
--- Running IR pass [map_matmul_to_mul_pass]
--- Running IR pass [fc_fuse_pass]
I0106 08:35:18.236121 23074 fuse_pass_base.cc:57] ---  detected 1 subgraphs
--- Running IR pass [repeated_fc_relu_fuse_pass]
--- Running IR pass [squared_mat_sub_fuse_pass]
--- Running IR pass [conv_bn_fuse_pass]
I0106 08:35:18.263962 23074 fuse_pass_base.cc:57] ---  detected 24 subgraphs
--- Running IR pass [conv_eltwiseadd_bn_fuse_pass]
--- Running IR pass [conv_transpose_bn_fuse_pass]
--- Running IR pass [conv_transpose_eltwiseadd_bn_fuse_pass]
--- Running IR pass [is_test_pass]
--- Running IR pass [runtime_context_cache_pass]
--- Running analysis [ir_params_sync_among_devices_pass]
--- Running analysis [adjust_cudnn_workspace_size_pass]
--- Running analysis [inference_op_replace_pass]
--- Running analysis [memory_optimize_pass]
I0106 08:35:18.290069 23074 memory_optimize_pass.cc:216] Cluster name : lstm_0.tmp_3  size: 1
I0106 08:35:18.290190 23074 memory_optimize_pass.cc:216] Cluster name : x  size: 384
I0106 08:35:18.290211 23074 memory_optimize_pass.cc:216] Cluster name : lstm_0._generated_var_0  size: 1
I0106 08:35:18.290228 23074 memory_optimize_pass.cc:216] Cluster name : relu_3.tmp_0  size: 2560
I0106 08:35:18.290239 23074 memory_optimize_pass.cc:216] Cluster name : linear_1.tmp_1  size: 26500
I0106 08:35:18.290251 23074 memory_optimize_pass.cc:216] Cluster name : lstm_0.tmp_1  size: 768
I0106 08:35:18.290261 23074 memory_optimize_pass.cc:216] Cluster name : hardswish_5.tmp_0  size: 1920
I0106 08:35:18.290271 23074 memory_optimize_pass.cc:216] Cluster name : batch_norm_27.tmp_3  size: 384
I0106 08:35:18.290282 23074 memory_optimize_pass.cc:216] Cluster name : hardsigmoid_8.tmp_0  size: 1152
--- Running analysis [ir_graph_to_program_pass]
I0106 08:35:18.406282 23074 analysis_predictor.cc:717] ======= optimize end =======
I0106 08:35:18.410946 23074 naive_executor.cc:98] ---  skip [feed], feed -> x
I0106 08:35:18.413607 23074 naive_executor.cc:98] ---  skip [softmax_0.tmp_0], fetch -> fetch
WARNING: Logging before InitGoogleLogging() is written to STDERR
I0106 08:35:18.413774 23074 main.cpp:215] The predict img: ../../doc/imgs/12.jpg
Detected boxes num: 1
The detection visualized image saved in ./ocr_vis.png
更唤谰	score: 0.000181636
更唤谰	score: 0.000181636
更唤谰	score: 0.000181636
更唤谰	score: 0.000181636
更唤谰	score: 0.000181636
更唤谰	score: 0.000181636

**Comment by andyjpaddle** [comment]
可否提供详细的运行命令以及所做的修改？

**Comment by wumuyu9** [comment]
> 可否提供详细的运行命令以及所做的修改？

谢谢，我已经找到原因了，应该是我将模型参数转为c++的时候出问题了，因为我重新下载模型再转就没这个问题了
